src/components/FilterDropdown.tsx

```diff
@@ -56,29 +61,36 @@ export default function FilterDropdown<T extends Named>({
     } else {
       setSelectedItems(selectedItems.filter((i) => i != item));
     }
-  }
+  };
 
   return (
     <>
       <div className="bg-red flex w-full flex-col">
-        <input
+        <div
           onChange={onChange}
-          className="h-[46px] w-full mb-2 px-3 border-2 border-tan focus:outline-none rounded"
-          onFocus={() => setShowOptions(true)}
-          onClick={() => setShowOptions(true)}
-        />
+          className="mb-2 h-[36px] w-full flex-row items-end justify-end rounded-md border-2 border-tan bg-white px-3 text-gray-400 focus:outline-none"
+          // onFocus={() => setShowOptions(true)}   @deprecated due to selector not supporting typing to search for student names
```

Comment:
nit: I don't think this should be marked `@deprecated`, since we're not getting rid of `FilterDropDown`. As an aside, annotations should be marked above the component itself. The idea is that when we hover the mouse over the component, we will get some additional information.